.github/workflows/sdist.yml

```diff
@@ -0,0 +1,56 @@
+name: sdist
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+      - 1.2.x
```

Comment:
we need a way to update this? i suppose manual is ok (but i think need to be in our release instructions).

cc @simonjayhawkins

Reply:
we could change the name of the branch to something like `minor` or `patch`.

Reply:
> we need a way to update this? i suppose manual is ok (but i think need to be in our release instructions).

yep its manual for now. eg. #41966. for the last release it was a replace. but since we haven't finished with 1.2.x yet, this time is an addition. 




> we could change the name of the branch to something like `minor` or `patch`.

generally this would only work if we finished with the backport branch before the rc. which we didn't do this time around.

Reply:
Maybe we could leave out the branches on some of the configs so that ci runs on all branches. It is only Web and Docs that we don't want to run on a actual push since that would update the dev docs so we only want that on master, but for a PR can run on any branch so that the doc build is tested.

Reply:
can you update to 1.3.x (or also add 1.2.x here as well)

Reply:
it's on the next line.